### PR TITLE
lime-proto-bmx7: update script and options

### PR DESCRIPTION
**Pull request by p4u**
Follows bmx6 changes.

Signed-off-by: p4u <p4u@dabax.net>

Still not tested in a real, production node. Waiting for comments...

### CODE REVIEW

File: packages/lime-proto-bmx7/src/bmx7.lua [hunk 1]
```diff
@@ -37,7 +39,7 @@ function bmx7.configure(args)
 	-- Enable JSON plugin to get bmx7 information in json format
 	uci:set(bmx7.f, "json", "plugin")
 	uci:set(bmx7.f, "json", "plugin", "bmx7_json.so")
-	
+
 	-- Enable SMS plugin to enable sharing of small files
```
Comment (aparcar) [inline packages/lime-proto-bmx7/src/bmx7.lua]: @axn does the SMS plugin add considerable overhead when unused? Maybe this should be configurable but default to true.

File: packages/lime-system/files/etc/config/lime-defaults [hunk 2]
```diff
@@ -17,6 +17,7 @@ config lime network
 	option anygw_dhcp_limit '0'
 	option main_ipv6_address '2a00:1508:0a%N1:%N200::/64'
 	option bmx6_mtu '1500'
+	option bmx7_mtu '1500'
```
Comment (aparcar) [inline packages/lime-system/files/etc/config/lime-defaults]: If 1500 is fallback there is no need to put it in lime-default but in lime-examples right?

**Comment by aparcar** [comment]
Tested `bmx7_mtu` and `bmx7_pref_gw` and looks good to me.

File: packages/lime-system/files/etc/config/lime-defaults (hunk 2 above)
Comment (p4u) [reply packages/lime-system/files/etc/config/lime-defaults]: No. By design all available options must exist in lime-defaults. In addition the MTU is not fallback, it is a needed parameter.

File: packages/lime-proto-bmx7/src/bmx7.lua (hunk 1 above)
Comment (p4u) [reply packages/lime-proto-bmx7/src/bmx7.lua]: AfAIK SMS plugin does nothing if it's not used.

**Pull request merged**